mesonbuild/cmake/traceparser.py

```diff
@@ -229,7 +230,8 @@ def parse(self, trace: T.Optional[str] = None) -> None:
             tgt.depends = tgtlist_gen(tgt.depends, tgt)
 
         for ctgt in self.custom_targets:
-            ctgt.outputs = pathlist_gen(ctgt.outputs)
+            ctgt.outputs = pathlist_gen(ctgt._outputs_str)
+            mlog.log(ctgt.outputs)
```

Comment:
This debug output is still present and will print to the console for regular users.

Reply:
Yeah, forgot about that one. I should really get more in the habit of checking my own commits.